contracts/CellarStaking.sol

```diff
@@ -555,6 +549,10 @@ contract CellarStaking is ICellarStaking, Ownable {
      * @param _paused               Whether the contract should be paused.
      */
     function setPaused(bool _paused) external override onlyOwner {
+        if (_paused == true && startTimestamp == 0) {
```

Comment:
shouldn't the check be the opposite? disallow unpausing if state is not initialized?

Reply:
Ah yeah, need to change to `_paused == false`